batch_seqr_loader/batch_workflow.py

```diff
@@ -670,15 +801,15 @@ def _add_haplotype_caller_job(
         f"""set -e
     (while true; do df -h; pwd; free -m; sleep 300; done) &
 
-    gatk --java-options "-Xms{java_mem}g -XX:GCTimeLimit=50 -XX:GCHeapFreeLimit=10" \
-      HaplotypeCaller \
-      -R {reference.base} \
-      -I {bam_fpath} \
-      -L {interval} \
-      -O {j.output_gvcf['g.vcf.gz']} \
-      -G StandardAnnotation -G StandardHCAnnotation -G AS_StandardAnnotation \
-      -GQB 10 -GQB 20 -GQB 30 -GQB 40 -GQB 50 -GQB 60 -GQB 70 -GQB 80 -GQB 90 \
-      -ERC GVCF \
+    gatk --java-options "-Xms{java_mem}g -XX:GCTimeLimit=50 -XX:GCHeapFreeLimit=10" \\
+      HaplotypeCaller \\
+      -R {reference.base} \\
+      -I {bam_fpath} \\
+      -L {interval} \\
+      -O {j.output_gvcf['g.vcf.gz']} \\
+      -G StandardAnnotation -G StandardHCAnnotation -G AS_StandardAnnotation \\
+      -GQB 10 -GQB 20 -GQB 30 -GQB 40 -GQB 50 -GQB 60 -GQB 70 -GQB 80 -GQB 90 \\
```

Comment:
Do we really need this fine-grained binning scheme? Just checking because we use something way more aggressive for TOB-WGS (inspired by gnomAD), but this is of course a different application.

I suspect that the KCCG reprocessing for the Acute Care samples has actually been done with the aggressive binning scheme, so this would be worth following up on and documenting why we chose a particular setting here.

Reply:
Oh, definitely not needed! Updated to just -GQB 20.